contracts/IProperty.sol

```diff
@@ -0,0 +1,7 @@
+pragma solidity ^0.4.20;
+
+import "openzeppelin-solidity/contracts/token/ERC721/ERC721Full.sol";
+import "./IERC721WithCreator.sol";
+
+contract IProperty is ERC721Full, IERC721WithCreator {
```

Comment:
I'd say lets actually have it an ```interface``` and ugly import all the functions we need. When will solidity allow interface inheritance @axic?????

Reply:
There's an issue for it. Hopefully soon.